examples/express/src/components/MyFragment.jsx

```diff
@@ -0,0 +1,34 @@
+// components/MyFragment.js
+import React from 'react';
+
+export default class MyFragment extends React.Component {
```

Comment:
Could we use functional components instead?

Reply:
Yes it's planned, I'll do it in an other PR